### PR TITLE
fix(progress-bar): [#FOR-807] wait elements saving before displaying pop up for a new one

### PR DESCRIPTION
**Pull request by qperie**
## Describe your changes
We add a loader to a pop up to wait that all the form elements are saved before displaying anything

## Issue ticket number and link
FOR-807 : https://jira.support-ent.fr/browse/FOR-807

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [ ] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: formulaire/src/main/resources/public/template/lightbox/new-element.html [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
+            <div class="dominos">
+                <div ng-repeat="type in vm.questionTypes.all" class="item"
+                     ng-click="vm.isProcessing || vm.doCreateNewElement(type.code, vm.parentSection)" title="[[vm.displayTypeDescription(type.name)]]">
+                    <div class="domino">
+                        <div class="data-text">
+                            <h5>[[vm.displayTypeName(type.name)]]</h5>
+                        </div>
+                        <div class="picture">
+                            <img ng-src="[[(vm.iconUtils.displayTypeIcon(type.code)) || '/formulaire/public/img/empty-image.png']]"/>
+                        </div>
```
Comment (qperie) [inline formulaire/src/main/resources/public/template/lightbox/new-element.html:19]: C'est le même code qu'avant, on l'a juste intégré à l'intérieur de la balise "<div  ng-if="!vm.isProcessing">"

File: formulaire/src/main/resources/public/ts/controllers/form-editor.ts [hunk 2]
```diff
@@ -122,7 +120,6 @@ export const formEditorController = ng.controller('FormEditorController', ['$sco
         vm.iconUtils = IconUtils;
 
         vm.$onInit = async () : Promise<void> => {
-            vm.initializing = true;
```
Comment (qperie) [inline formulaire/src/main/resources/public/ts/controllers/form-editor.ts:125]: unused

File: formulaire/src/main/resources/public/ts/controllers/form-editor.ts [hunk 3]
```diff
@@ -132,7 +129,6 @@ export const formEditorController = ng.controller('FormEditorController', ['$sco
             $scope.safeApply();
 
             initNestedSortables();
-            vm.initializing = false;
```
Comment (qperie) [inline formulaire/src/main/resources/public/ts/controllers/form-editor.ts:135]: unused

File: formulaire/src/main/resources/public/ts/controllers/form-editor.ts [hunk 4]
```diff
@@ -65,7 +64,6 @@ interface ViewModel {
     PreviewPage: typeof PreviewPage;
     nestedSortables: any[];
     iconUtils: IconUtils;
-    initializing: boolean;
```
Comment (qperie) [inline formulaire/src/main/resources/public/ts/controllers/form-editor.ts:68]: unused

**Review by Rseuret — APPROVED** [review]

**Pull request merged**